content/v3/pulls/comments.md

```diff
@@ -62,7 +62,7 @@ Name | Type | Description
 Name | Type | Description
 -----|------|--------------
 `body`|`string` | **Required**. The text of the comment
-`commit_id`|`string` | **Required**. The SHA of the commit to comment on.
+`commit_id`|`string` | **Required**. The SHA of the latest commit of the pull request to comment on.
```

Comment:
> If it is the line in the diff, then that is a bit hard to figure out when using the API. Perhaps the files response can be extended to help translating a line number in the file to a line number in the diff.

One solution may be to [use the `application/vnd.github.diff` or `application/vnd.github.patch` media types](https://developer.github.com/changes/2012-12-10-Diff-and-patch-media-types/).

Or, in the browser, by adding `.diff` or `.patch` to the end of the URL. For example:
- https://github.com/github/developer.github.com/pull/1020.diff
- https://github.com/github/developer.github.com/pull/1020.patch